2-structured-data/test/_crud-tests.js

```diff
@@ -13,180 +13,142 @@
 
 'use strict';
 
-const getRequest = require(`@google-cloud/nodejs-repo-tools`).getRequest;
-const test = require(`ava`);
+const getRequest = require('@google-cloud/nodejs-repo-tools').getRequest;
+const assert = require('assert');
 
 module.exports = DATA_BACKEND => {
   let originalDataBackend, id, testConfig, appConfig;
 
-  test.before(() => {
-    testConfig = require(`./_test-config`);
-    appConfig = require(`../config`);
-    originalDataBackend = appConfig.get(`DATA_BACKEND`);
-    appConfig.set(`DATA_BACKEND`, DATA_BACKEND);
+  before(() => {
+    testConfig = require('./_test-config');
```

Comment:
Is there any directive to use single quote in place of backtick symbol?  If not I suggest you to use backtick only as it won;'t create unnecessary lines change

Reply:
Changed as per suggestion in previous review.